activerecord/test/cases/transaction_callbacks_test.rb

```diff
@@ -706,18 +708,18 @@ def test_callbacks_run_in_order_defined_in_model_if_using_run_after_transaction_
 
     topic.clear_history
     topic.destroy
-    assert_equal [1, 2, 3, 4, "destroy"], topic.history
+    assert_equal [1, 2, "prepend 2", "prepend 1", 3, 4, "destroy"], topic.history
   end
 
-  def test_callbacks_run_in_order_defined_in_model_if_not_using_run_after_transaction_callbacks_in_order_defined
+  def test_callbacks_run_in_reverse_order_defined_in_model_if_not_using_run_after_transaction_callbacks_in_order_defined
```

Comment:
Pretty sure this test name is incorrect, it should be the opposite of the test above. If not using the in-order-config they should be running in reverse order. The assertions agree with this as well.